web-jehlomat/src/utils/payload-status.ts

```diff
@@ -0,0 +1,19 @@
+type TStatus = number | string
+
+export const isStatusGeneralSuccess = (status: TStatus) =>
+  /2[0-9][0-9]/g.test(status.toString())
+
+export const isStatusSuccess = (status: TStatus) =>
+  status.toString() === "200"
+
+export const isStatusValidationError = (status: TStatus) =>
```

Comment:
Personally, I would expect status code 400. Status code 409 is a conflict, so a user tries to create a duplicate record. Consider renaming.